### ISSUE TITLE
support for insecure TLS 

### ISSUE THREAD

**Issue by hellt** [p4lang/p4runtime-shell#101]
Hi all,

would it be possible to implement an `--skip-tls-verify` option that will allow TLS certificates but won't require a user to provide a CA file to validate the certificate?
It is cumbersome to work with network OSes that mandate the use of TLS for gRPC in lab scenarios where certification verification is not a concern.

**Comment by antoninbas** [comment]
I don't know if this is possible with the Python gRPC library.
If it is possible, feel free to submit a PR with your proposed changes.

**Comment by hellt** [comment]
Yes, it seems you're right; the skip-verify callback is not available in grpc-py...
I will check if p4-go client can satisfy my needs, thanks.
